### DISCUSSION TITLE
Unable to flash firmware on Yi Outdoor

### DISCUSSION THREAD

**Discussion by AngryMooses** [alienatedsec/yi-hack-v5#240] [Q&A]
I'm unable to access the web admin page, connecting http://10.0.4.254:8080 (the IP address of my camera), and I'm not sure if the firmware update worked.  Any guidance?

Here's what I've done so far:
- Format a 32GB SD card as FAT32 on my Mac (don't have a windows device)
- Copied over the 0.3.7 files including the yi-hack-v5 directory
- Insert card into camera, power on
- Yellow light comes on, then blue, but no second yellow light as indicated in the readme
- Unable to access the web admin page
- Check card contents, and new directories have been written to the card by the camera so it must be mounting the card correctly.  Verified rootfs_h30 and home_h30 haven't been renamed.
- Reformat card, put firmware version 0.3.6 files on there
- Same result
- Update firmware using Yi app (shipped with 9.x, latest is 11.x)
- Attempted update again, with 0.3.7
- Same result

**Comment by alienatedsec** [comment]
Wrong camera model

**Reply by nanock91** [reply]
I'm having the same problem. I bought the Xiaomi home 1080p through amazon (https://www.amazon.es/dp/B01MR82NFZ) and I am using the correct firmware. When turning on the camera with the sd card inserted, it goes from yellow to blue and does not perform the installation process. The SD is completely new, with a 4gb FAT32 partition and the files inside.

Why can this happen?

On QR background said: RFUSY62DH6 1080p
